test/MockSmartSyncPlugin.js

```diff
@@ -103,10 +103,13 @@ var MockSmartSyncPlugin = (function(window) {
             var progress = 0;
             collection.cache = cache;
 
-            // Resync?
-            var maxTimeStamp = sync.maxTimeStamp;
-            if (target.type == "soql" && _.isNumber(maxTimeStamp) && maxTimeStamp > 0) {
-                collection.config = {type:"soql", query: self.addFilterForReSync(target.query, maxTimeStamp)};
+            if (target.type == "soql") {
+                // Missing Id or LastModifiedDate
+                var query = self.addSelectFieldIfMissing(self.addSelectFieldIfMissing(target.query, "Id"), "LastModifiedDate");
```

Comment:
On the native side, we add Id and LastModifiedDate, so the doing the same thing with the mock.
Before SmartSyncExplorerHybrid was specifying Id and LastModifiedDate in its sync down soql. But we want to use the exact usersyncs.json.